### PR TITLE
Remove Sphinx as a required dependency

### PR DESCRIPTION
**Pull request by phlax**

### CODE REVIEW

**Comment by betaboon** [comment]
is there anything missing to get this in?
We're also running into this problem.

**Comment by phlax** [comment]
> is there anything missing to get this in?

+1

**Comment by betaboon** [comment]
@AA-Turner friendly bump

**Comment by andy-maier** [comment]
Gentle reminder to get this in. We are also hampered by the circular dependency (using pipdeptree).

**Comment by phlax** [comment]
for bazel users it appears there is a workaround - see https://github.com/bazelbuild/rules_python/pull/1586

struggling to understand why this fix was not just landed - a waste of my time - and tbh the workaround should not be necessary

File: CHANGES [hunk 1]
```diff
@@ -1,3 +1,8 @@
+Release 1.1.10 (Pending)
+==========================
```
Comment (AA-Turner) [inline CHANGES]:
```suggestion
Release 1.1.10 (unreleased)
===========================
```

File: pyproject.toml [hunk 2]
```diff
@@ -54,6 +51,9 @@ lint = [
     "mypy",
     "docutils-stubs",
 ]
+standalone = [
+  "sphinx > 5",
```
Comment (AA-Turner) [inline pyproject.toml]:
```suggestion
    "Sphinx>=5",
```

File: CHANGES [hunk 3]
```diff
@@ -1,3 +1,8 @@
+Release 1.1.10 (unreleased)
+===========================
+
+* Fix circular dependency with Sphinx that caused failure in DAG-based package management
```
Comment (AA-Turner) [inline CHANGES]:
```suggestion
* Remove Sphinx as a required dependency, as circular dependencies may cause
  failure with package managers that expect a directed acyclic graph (DAG)
  of dependencies.
```

File: pyproject.toml [hunk 4]
```diff
@@ -40,9 +40,6 @@ classifiers = [
     "Topic :: Text Processing",
     "Topic :: Utilities",
 ]
-dependencies = [
-    "Sphinx>=5",
-]
 dynamic = ["version"]
```
Comment (AA-Turner) [inline pyproject.toml:44]:
```suggestion
dependencies = []
dynamic = ["version"]
```

**Pull request merged**